CameraOverlay/commons.py

```diff
@@ -1,15 +1,20 @@
 import os
 import json
+import fnmatch
 from dotenv import load_dotenv
 
-from .topics import *
-
 load_dotenv()
 CONFIG_FILE = 'configs.json'
 ACTIVE_OVERLAY_KEY = 'activeOverlay'
+OVERLAY_FILE_PATTERN = 'Overlay_Demo*.py'
```

Comment:
Maybe the pattern should just be `Overlay_*.py`? Not all our overlays will be demos in the future

Reply:
Yeah, the naming convention of the camera overlays should be changed. @TienMinhNguyen just left it when we started them